File: Homeworks/C# Part 2/03.Methods/02.GetLargestNumber/GetLargestNumber.cs
using System;

class GetLargestNumber
{
    static int GetMax(int firstNumber, int secondNumber)
    {
        return Math.Max(firstNumber, secondNumber);
    }

    static void Main()
    {
        string[] input = Console.ReadLine().Split(' '); 
        int numberA = int.Parse(input[0]);
        int numberB = int.Parse(input[1]);
        int numberC = int.Parse(input[2]);
        Console.WriteLine(GetMax(GetMax(numberA, numberB), numberC));
    }
}

File: Homeworks/JS Fundamentals/08. Functions/02. Get largest number/get-largest-number.js
function solve(args) {
    var numbers = args[0].split(' ');
    var max;

    function getMax(first, second) {
        return Math.max(first, second);
    }
    max = getMax(+numbers[0], +numbers[1]);
    max = getMax(max, +numbers[2]);
    console.log(max);
}